edx-platform/pearson-vue-theme/lms/templates/ccx/enrollment.html

```diff
@@ -116,6 +116,19 @@ <h2 class="hd hd-2">${_("Student List Management")}</h2>
                 %endif
               </tr>
             %endfor
+            %if pending_ccx_members:
+              %for pending_enrollment in pending_ccx_members:
+                <tr>
+                  <td>${pending_enrollment.get('user_name')}</td>
+                  <td>${pending_enrollment.get('user_email')}</td>
+                  <td>
+                    <button type="button" class="revoke">
+                      <span class="fa fa-times-circle" aria-hidden="true"></span>${_("Revoke access")}
+                    </button>
+                  </td>     
+                </tr>
+              %endfor
+            %endif
```

Comment:
Isn't this equivalent? . @anfbermudezme 
```suggestion
            %for pending_enrollment in pending_ccx_members:
              <tr>
                <td>${pending_enrollment.get('user_name')}</td>
                <td>${pending_enrollment.get('user_email')}</td>
                <td>
                  <button type="button" class="revoke">
                    <span class="fa fa-times-circle" aria-hidden="true"></span>${_("Revoke access")}
                  </button>
                </td>     
              </tr>
            %endfor

```

Reply:
No, because if the run_extension_fails for example:

![image](https://user-images.githubusercontent.com/30726391/188213619-1545d35a-3196-4973-b418-a1d6d56c30db.png)

The error that appears is:

![image](https://user-images.githubusercontent.com/30726391/188213814-16e9b6a0-a571-4fab-974d-30b8fe248a68.png)